packages/web/src/components/toast/ToastContext.tsx

```diff
@@ -34,7 +34,7 @@ type ToastContextProps = {
   clear: () => void
 }
 
-type Toast = {
+type ToastType = {
```

Comment:
I prefer not having `Type` in the name of a type, kinda feels like it denotes an enum of different variants of a toast in this case